var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -101,7 +101,7 @@ def patch(self):
 
     @property
     def common_config_args(self):
-        return [
+        config_args = [
             '-prefix', self.prefix,
```

Comment:
Yep, but version 3 isn't using common_config_args, so it isn't specifying the prefix. That's what I meant.